tests/Support/SupportHelpersTest.php

```diff
@@ -552,7 +552,7 @@ public function testRetry()
         $this->assertEquals(2, $attempts);
 
         // Make sure we waited 100ms for the first attempt
-        $this->assertEqualsWithDelta(0.1, microtime(true) - $startTime, 0.02);
+        $this->assertEqualsWithDelta(0.1, microtime(true) - $startTime, 0.03);
```

Comment:
microtime is not appropriate to use for timing. hrtime needs to be used, otherwise these tests will be flakey.